src/utils/github.ts

```diff
@@ -94,15 +94,17 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | null> {
     logger.info({ path }, 'GitHub API: Get Content');
-    const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return Buffer.from(response.data.content, 'base64').toString();
-    } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
-    }
+    const response = await this.octokit.rest.repos
+      .getContent({ owner: this.owner, repo: this.repo, path })
+      .catch((e) => {
+        if (e.status === 404) return null;
```

Comment:
```suggestion
        if (e?.status === 404) return null;
```

in theory e could be null here, very unlikely but could happen.